packages/docmanager-extension/src/index.ts

```diff
@@ -181,29 +195,73 @@ function addCommands(app: JupyterLab, docManager: IDocumentManager, palette: ICo
     const fts = docRegistry.getFileTypesForPath(context.path);
     return (fts.length && fts[0].displayName) ? fts[0].displayName : 'File';
   };
+  const closeWidgets = (widgets: Array<Widget>): void => {
+    widgets.forEach(widget => { widget.close(); });
+  };
+  const widgetsRightOf = (widget: Widget): Array<Widget> => {
+    const { id } = widget;
+    const widgets = toArray(shell.widgets('main'));
+    const index = widgets.findIndex( widget => widget.id === id );
```

Comment:
Please format this without spaces inside the parens:
`const index = widgets.findIndex(widget => widget.id === id);`
We only use spaces in object literals: `{ foo: 'bar' }`